docs/alice.md

```diff
@@ -0,0 +1,64 @@
+`relay alice`
```

Comment:
Do we want to publish an `alice` doc?

Reply:
Whoops, probably not yet - still need to work through the alice commands